src/main/java/storage/ParseDataFile.java

```diff
@@ -0,0 +1,87 @@
+package storage;
+
+import cheatsheet.CheatSheet;
+import cheatsheet.CheatSheetList;
+import exception.InvalidFileDataException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class ParseDataFile {
```

Comment:
A few of the class names sound more like method names. Perhaps instead of ParseDataFIle, DataFileParser

Reply:
Noted! Will rectify them.